### DISCUSSION TITLE
Default value in multicascader

### DISCUSSION THREAD

**Discussion by Boopalan-M** [rsuite/rsuite#2018] [General]
How can we set the default value in multicascader. I passed the value from props to defaultValue.But it doesn't set.

**Comment by simonguo** [comment]
@Boopalan-M  I tried it and it works fine. Please use codesandbox to reproduce your problem.
![image](https://user-images.githubusercontent.com/1203827/138225968-c227c321-e250-4279-af69-09cb1f647a51.png)
